Trustroots/trustroots
# Add script for populating test DB with experiences

fixes  [#1699](https://github.com/Trustroots/trustroots/issues/1669)

#### Testing instructions

- run `npm seed`
- check the DB is populated with references
